ravenframework/Models/PostProcessors/Validations/PPDSS.py

```diff
@@ -124,10 +124,10 @@ def _handleInput(self, paramInput):
       if child.getName() == 'Metric':
         if 'type' not in child.parameterValues.keys() or 'class' not in child.parameterValues.keys():
           self.raiseAnError(IOError, 'Tag Metric must have attributes "class" and "type"')
-      elif child.getName() == 'Features':
-        self.features = child.value
-      elif child.getName() == 'Targets':
-        self.targets = child.value
+      elif child.getName() == 'prototypeOutputs':
+        self.prototypeOutputs = child.value
+      elif child.getName() == 'targetOutputs':
+        self.targetOutputs = child.value
```

Comment:
base class can be called to read these nodes, such as ```super()._handleInput(paramInput)```

Reply:
This is already there. I am not sure I understood your suggestion. @wangcj05